note

```diff
@@ -1,29 +1,38 @@
 #!/bin/bash
 
-create_note () {
-    if [ ! -f $note_path ]; then
-        mkdir -p ~/notes/$year/$month/
-        touch $note_path
-        printf "$day-$month-$year\n---\n\n" > $note_path
-        printf "Created new note: $note_path\n"
+YEAR=$(date +'%Y')
+MONTH=$(date +'%m')
+DAY=$(date +'%d')
+NOTE_DIR="$HOME/notes/$YEAR/$MONTH"
+NOTE_NAME="$DAY.md"
+
+NOTERC="$XDG_CONFIG_HOME/note/noterc"
```

Comment:
I know that `$XDG_CONFIG_HOME` does not exist in MacOS. I'd like to maintain full compatibility for Mac, so maybe we need to add logic that says "if `$XDG_CONFIG_HOME` does not exist, default to `~/.config/notekeeper/noterc`"

Also although the executable is `note` the app name is `notekeeper` so should probably be `$XDG_CONFIG_HOME/notekeeper/noterc`.